numpy/lib/npyio.py

```diff
@@ -980,40 +980,33 @@ def loadtxt(fname, dtype=float, comments='#', delimiter=None,
 
     def split_line(line):
```

Comment:
```suggestion
    def split_line(line: str):
```
would hopefully make it clear that `line` is already decoded

Reply:
done